src/ChplPaths.ts

```diff
@@ -195,3 +195,50 @@ export function getWorkspaceFolder(): vscode.WorkspaceFolder | undefined {
   const doc = editor.document;
   return vscode.workspace.getWorkspaceFolder(doc.uri);
 }
+
+export function getEnvAffectingChapel(): Map<string, string> {
```

Comment:
A small comment about what this function does and in which cases it's still the user's responsibility to set these variables in their debug configs would be helpful for the future.